### DISCUSSION TITLE
Reduce jpeg file size

### DISCUSSION THREAD

**Discussion by kemalgencay** [ImageMagick/ImageMagick#4545] [Digital Image Processing]
I could not spot a command that reduces the file size of a jpeg image from about 2-3 MB down to about 20-30 kb to load on web page.
Any help please
TIA
Kemal

**Comment by fmw42** [comment]
See -quality or -define jpeg:extent={size}  See https://legacy.imagemagick.org/Usage/formats/#jpg_write
